Add community days 2025

vkrasnovyd opened the pull request:
* Added Community Days 2025 with content similar to CFP-2024 and new location and dates
* Since It seems to be the normal workflow to add CFP first and replace page content with the program later, update the community days archetype (for generating new page)
* Made button shortcode customizeable and added a new styling shortcode, described both in the docs
* Enabled rendering button in the event card in /events/, described usege in the docs. Url and text for the button are defined in the markdown-file for the event. Archetype contains default values. If one or both values are missing, an error is rendered. If the whole 'button' block is missing in the markdown-file, the button is simply not rendered, no errors are displayed.

**To be done:** add notification on the main page about the event and link to the page.

vkrasnovyd commented on archetypes/community-days.md:
```diff
@@ -14,81 +15,80 @@ params:
+Please submit your presentation proposal by clicking on the link below:
 
-### Welcome & Keynote
+{{< register-button email="cfp@csaf.io" text="Submit Your Proposal" >}}
```
Added like in the example.

vkrasnovyd commented on archetypes/community-days.md:
```diff
@@ -14,81 +15,80 @@ params:
       display_in_lists: true
       display_on_top: true
+      button:
+        url: 'mailto:cfp@csaf.io'
```
Added for both page and archetype as optional parameter url_params. Technically it can be used for either setting subject for emails or adding custom parameters to the page urls.

tschmidtb51 commented on archetypes/community-days.md:
Hmm. Then, we should actually have one for the CFP and one for the schedule...

vkrasnovyd commented on archetypes/community-days.md:
```diff
@@ -14,81 +15,80 @@ params:
 
-{{< /session-card >}}
+If you have any questions, feel free to reach out to us at
+[csaf@bsi.bund.de](mailto:csaf@bsi.bund.de).
```
Is subject "[CSAF Community Days 2025]" sufficient in this case?

vkrasnovyd replied:
Updated base email. Waiting for the answer about the subject.

vkrasnovyd commented on archetypes/community-days.md:
```diff
@@ -14,81 +15,80 @@ params:
-email to [csaf@bsi.bund.de](mailto:csaf@bsi.bund.de). We look forward to
-welcoming you to Munich!
+`To learn how to customize and style an event page, please see the instructions in the README file:`
+https://github.com/csaf-auxiliary/csaf-website-relaunch#add-a-page-for-the-event
```
Fixed in all the archetypes.

vkrasnovyd commented on content/community-days/2025.md:
```diff
@@ -0,0 +1,85 @@
+
+### Event Location & Hybrid Format
+
+The CSAF Community Days 2024 will be held as a hybrid event, allowing
```
Fixed in 4720c16

vkrasnovyd commented:
Along with small improvements, added to the home page the optional highlighted link to the selected page:
<img width="1918" height="957" alt="image" src="https://github.com/user-attachments/assets/b9a4d064-017d-4f87-9175-cce351c120c2" />

Will update the documentation with the information how to change/remove this block.

bernhardreiter commented on README-repo.md:
```diff
@@ -47,32 +48,32 @@ Before starting the development server, ensure you have the following installed:
   Required for generating the static website from the source code files.
 
-    - Check if Hugo is installedby running `hugo version` in the terminal.
+  - Check if Hugo is installedby running `hugo version` in the terminal.
```
`installedby` -> `installed, by`

vkrasnovyd replied:
Fixed.

bernhardreiter commented on archetypes/community-days-cfp.md:
```diff
@@ -0,0 +1,95 @@
+`Values in backticks below (dates and location) are the placeholders. Update them and remove the backsticks before publishing.`
+
+`To learn how to customize and style an event page, please see the instructions in the README file:`
+https://github.com/oasis-open/csaf-documentation/main/README-repo.md#add-a-page-for-the-event
```
Maybe use a relative link?

vkrasnovyd replied:
Fixed for all the archetypes.

bernhardreiter commented:
In my mindset, the card on `/events/` should not have a _submit_ button that directly tries to send an email. I've just clicked on it and wanted to get to the details.

When pressing on the banner message which goes to `/events/community-days/2025/` I get a page not found message.

vkrasnovyd commented on docs/index.md:
```diff
@@ -0,0 +1,39 @@
+# Changing the highlighted link
+
+When a page should be promoted to the site visitors (for example new event),
+a banner can be added on the homa page right below the navigation bar.
```
Fixed.

vkrasnovyd commented:
> In my mindset, the card on `/events/` should not have a _submit_ button that directly tries to send an email. I've just clicked on it and wanted to get to the details.

Replaced the button with just an optional subtitle as discussed yesterday.

> When pressing on the banner message which goes to `/events/community-days/2025/` I get a page not found message.

Fixed.

> Hmm we should have the large card on `community-days` as well, I think.

Done. Now all the events list pages have the same layouts, with big cards.

> ` [cfp@csaf.io](mailto:cfp@csaf.io?subject=[CSAF Community Days 2025])`. does not work

Fixed.

> I think we should also have the banner on all subpages, except the `events` page where there is the large card.

Work in progress, will be finished in the afternoon.

bernhardreiter commented:
@justmurphy the work is done from our (Intevation's) side. So it is over to you. 

Note: some minor general minor layout problems are know, e.g in the https://github.com/csaf-auxiliary/csaf-website-relaunch/issues and are left for  later as the current state is good enough (from what we can say) and should go online as soon as it can.

santosomar commented:
A motion to approve the creation of a CFP and implement additional updates to the CSAF website, as outlined in this PR, was proposed by Thomas S. and seconded by Stefan Hagen. The motion passed unanimously without discussion or objection.
